pages/platform/databases/mysql_08_advanced_parameters_references/guide.en-gb.md

```diff
@@ -0,0 +1,84 @@
+| `mysql.innodb_print_all_deadlocks` | boolean | When enabled, information about all deadlocks in InnoDB user transactions is recorded in the error log. Disabled by default. |
+| `mysql.innodb_read_io_threads` | long | The number of I/O threads for read operations in InnoDB. Default is 4. Changing this parameter will lead to a restart of the MySQL service. |
+| `mysql.innodb_rollback_on_timeout` | boolean | When enabled a transaction timeout causes InnoDB to abort and roll back the entire transaction. Changing this parameter will lead to a restart of the MySQL service. |
+| `mysql.innodb_thread_concurrency` | long | Defines the maximum number of threads permitted inside of InnoDB. Default is 0 (infinite concurrency - no limit) |
+| `mysql.innodb_write_io_threads` | long | The number of I/O threads for write operations in InnoDB. Default is 4. Changing this parameter will lead to a restart of the MySQL service. |
+| `mysql.interactive_timeout` | long | The number of seconds the server waits for activity on an interactive connection before closing it. |
+| `mysql.internal_tmp_mem_storage_engine` | string | The storage engine for in-memory internal temporary tables. |
+| `mysql.long_query_time` | double | The slow_query_logs work as SQL statements that take more than long_query_time seconds to execute. Default is 10s |
+| `mysql.max_allowed_packet` | long | Size of the largest message in bytes that can be received by the server. Default is 67108864 (64M) |
+| `mysql.max_heap_table_size` | long | Limits the size of internal in-memory tables. Also set tmp_table_size. Default is 16777216 (16M) |
+| `mysql.net_buffer_length` | long | Start sizes of connection buffer and result buffer. Default is 16384 (16K). Changing this parameter will lead to a restart of the MySQL service. |
+| `mysql.net_read_timeout` | long | The number of seconds to wait for more data from a connection before aborting the read. |
+| `mysql.net_write_timeout` | long | The number of seconds to wait for a block to be written to a connection before aborting the write. |
+| `mysql.slow_query_log` | boolean | Slow query log enables capturing of slow queries. Setting slow_query_log to false also truncates the mysql.slow_log table. Default is off |
+| `mysql.sort_buffer_size` | long | Sort buffer size in bytes for ORDER BY optimization. Default is 262144 (256K) |
+| `mysql.sql_mode` | string | Global SQL mode. Set to empty to use MySQL server defaults. When creating a new service and not setting this field Aiven default SQL mode (strict, SQL standard compliant) will be assigned. |
+| `mysql.sql_require_primary_key` | boolean | Require primary key to be defined for new tables or old tables modified with ALTER TABLE and fail if missing. It is recommended to always have primary keys because various functionality may break if any large table is missing them. |
```

Comment:
```suggestion
| `mysql.sql_require_primary_key` | boolean | Require primary key to be defined for new tables or old tables modified with ALTER TABLE and fail if missing. It is recommended to always have primary keys because various functionalities may break if any large table is missing them. |
```